src/main/groovy/org/shipkit/gradle/exec/ShipkitExecTask.java

```diff
@@ -0,0 +1,39 @@
+package org.shipkit.gradle.exec;
+
+import org.gradle.api.DefaultTask;
+import org.gradle.api.tasks.TaskAction;
+import org.shipkit.internal.gradle.exec.ShipkitExec;
+
+import java.util.Collection;
+import java.util.LinkedList;
+
+/**
+ * Generic Gradle task that allows composing multiple executable command line invocations.
+ * Very useful for scenarios where we cannot easily use multiple Gradle Exec tasks.
+ */
+public class ShipkitExecTask extends DefaultTask {
```

Comment:
I liked the name of the previous version (CompositeExecTask) better, I think it said more about the functionality of this Task.